Documentation/Migration/Migration_Install.md

```diff
@@ -0,0 +1,52 @@
+# Migrate an ACPCore iOS app to the latest AEPCore
+
+Existing ACPCore users who are interested in upgrading to the latest AEPCore SDK, this guide describes how to upgrade. We have made upgrading seamless by providing a backward-compatible layer, which will ensure that an existing ACPCore implementation will be compatible with AEPCore.
+
+## Upgrading
+
+### Cocoapods
+
+The easiest way to upgrade from ACPCore to AEPCore is by using Cocoapods. To upgrade, update the reference to `ACPCore` in your projects `Podfile` as such:
+
+```diff
+- pod 'ACPCore'
++ pod 'ACPCore', :git => 'https://github.com/adobe/aep-sdk-compatibility-ios.git', :branch => 'main'
+```
+
+ Then run `pod install` and build your project, and you will be using the latest AEPCore SDK. Under the hood, this new podspec brings in the latest AEPCore SDK via Cocoapods.
+
+#### Version Control
+
+To control which version of AEPCore will be used when using this upgrade method, specify the versions in the `Podfile`.
+
+```ruby
+pod 'AEPServices', '1.0.0'
```

Comment:
are we bundling the core extensions with the AEPCore cocoapod?  also should be version `3.0.0`

Reply:
As of now, we are not. If we decide to bundle them we can update this doc.